src/components/AuthorsSection.tsx

```diff
@@ -90,9 +94,11 @@ const AuthorsSection: React.FC<Props> = ({
                       {contributer[WITH_CONTRIBUTIONS_FIELD]}
                     </td>
                   )}
-                  <td align={ALIGN_TD}>
-                    {contributer[WITH_NUMBER_OF_REPOS_FIELD]}
-                  </td>
+                  {config.AuthorConfigs.AuthorInfo.WithNumberOfRepos && (
```

Comment:
Should probs use optional changing for safety unless these access keys are guaranteed 
Ex: A?.B instead of A.B